editor/src/main/com/mbrlabs/mundus/editor/ui/modules/dock/assets/AssetsDock.kt

```diff
@@ -137,6 +142,14 @@ class AssetsDock : Tab(false, false),
             }
         })
 
+        openDirectoryAsset.addListener(object : ClickListener() {
+            override fun clicked(event: InputEvent, x: Float, y: Float) {
+                currentSelection?.asset?.let {
+                    Desktop.getDesktop().open(it.file.file().parentFile)
```

Comment:
Why not use vis-ui's file chooser? https://github.com/kotcrab/vis-ui/wiki/File-chooser

Reply:
This is not for choosing a file, but opening the assets directory on your operating system.